src/main/java/frc/robot/Constants.java

```diff
@@ -234,4 +235,33 @@ public static final class Camera {
     /** The camera id for the intake camera. */
     public static final int INTAKE_CAMERA_ID = 0;
   }
+
+  /** The constants for the light strips */
+  public static class LightStrips {
+    /** The PWM ID for the underglow light strip */
+    public static final int UNDERGLOW_PWM_ID = 0;
+
+    /** The count of LEDs for the underglow light strip. */
+    public static final int UNDERGLOW_LED_COUNT = 150;
+
+    /** The speed of the fade animation. [0, 1] */
+    public static final double STRIP_FADE_SPEED = 0.05;
+
+    public static final class Colors {
```

Comment:
I would add some comments (NOT JAVADOC) in here specifying what color they actually are (maybe on the end of the line?).